.env.example

```diff
@@ -1,8 +1,21 @@
-export CHROMATIC_PROJECT_TOKEN=secret
-export WEB3STORAGE_TOKEN=secret
-export FAUNADB_SECRET=secret
-export RINKEBY_RPC_PATH=alchemypath
-export MAINNET_RPC_PATH=alchemypath
-export PINATA_PUBLIC_API_KEY=secret
-export PINATA_PRIVATE_API_KEY=secret
-export GITHUB_CLIENT_SECRET=secret
+NEXT_PUBLIC_LVL_V1_CONTRACT_ADDRESS=address
+NEXT_PUBLIC_NETWORK_ID=4
+NEXT_PUBLIC_ORIGIN=http://localhost:8080
+NEXT_PUBLIC_SITE_TITLE='Level Protocol'
+NEXT_PUBLIC_SITE_DESCRIPTION='lvl is a crypto resume: Level is an on-chain reputation and skills web3 resume that highlights all of your contributions across communities, DAOs, and metaverses'
+NEXT_PUBLIC_TWITTER_USERNAME='lvlprotocol'
+NEXT_PUBLIC_ALCHEMY_KEY=key
+NEXT_PUBLIC_FAUNA_URL=https://graphql.us.fauna.com/graphql
+NEXT_PUBLIC_GITHUB_OAUTH_CLIENT_ID=client_id
```

Comment:
do we really need these if we are already falling back to `.env.development` ?

Reply:
@royriver23 and I chatted about this and I’ve updated the `.env.example` with our resolved solution. :)